bamboo/create-release.sh

```diff
@@ -5,26 +5,19 @@ source .bamboo_env_vars || true
 . ./bamboo/set-bamboo-env-variables.sh
 . ./bamboo/abort-if-not-pr.sh
 
+# Build TF module artifacts
+. ./bamboo/create-release-artifacts.sh
+
 export VERSION=$(jq --raw-output .version lerna.json)
 
-## Create Release
-   export RELEASE_URL=$(curl -H\
+# Create Release
+export RELEASE_URL=$(curl -H\
   "Authorization: token $GITHUB_TOKEN"\
    -d "{\"tag_name\": \"v$VERSION\", \"target_commitsh\": \"v$VERSION\", \"name\": \"v$VERSION\", \"body\": \"Release v$VERSION\" }"\
    -H "Content-Type: application/json"\
    -X POST\
    https://api.github.com/repos/nasa/cumulus/releases |grep \"url\" |grep releases |sed -e 's/.*\(https.*\)\"\,/\1/'| sed -e 's/api/uploads/')
 
-## Build TF modules that require source building
-
-(cd tf-modules/distribution && ./bin/build-tf-module.sh && cp ./dist/terraform-aws-cumulus-distribution.zip ../../terraform-aws-cumulus-distribution.zip)
-(cd tf-modules/s3-replicator && ./bin/build-tf-module.sh && cp ./dist/terraform-aws-cumulus-s3-replicator.zip ../../terraform-aws-cumulus-s3-replicator.zip)
-
-## Create zipfiles
-zip -FS -r -x \*node_modules\* \*internal\* -o terraform-aws-cumulus.zip tf-modules lambdas tasks/**/dist/** packages/**/dist/**
-(cd ./tf-modules/workflow; zip -FS -r -x \*node_modules\* -o ../../terraform-aws-cumulus-workflow.zip .)
-(cd ./tf-modules/cumulus_ecs_service; zip -FS -r -x \*node_modules\* -o ../../terraform-aws-cumulus-ecs-service.zip .)
```

Comment:
Why did you remove these?

Reply:
I moved them to the new `create-release-artifacts.sh` script, because when you want to do local testing of the next release, you want to run that script to create the artifacts locally without the rest of the behavior of the `create-release.sh` script